### DISCUSSION TITLE
Doris State table does not exist when use cloud mode 3.x

### DISCUSSION THREAD

**Discussion by fdf1779** [apache/doris#43717] [A - General / Q&A]
When I use Doris 3's storage computation separation mode, Doris' FE logs continue to print INFO [StatisticUtil. statsTblAvailable(): 494] stat table does not exist, db_name: __internal_schema, table_name: column_statistics。 If the built-in table is no longer needed, can we reduce the printing of logs or stop printing such logs. Why didn't create a built-in table if needed

**Comment by parkour99** [comment]
you can check the be entrypoint script.
if the be has registered successfully, then fe will create column_statistics automatically.
